src/strategies/stablexswap/examples.json

```
@@ -0,0 +1,21 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "stablexswap",
+      "params": {
+        "address": "0x0da6ed8b13214ff28e9ca979dd37439e8a88f6c4",
+        "symbol": "STAX",
+        "decimals": 18
+      }
+    },
+    "network": "42",
+    "addresses": [
+      "0x10f09b9942707cea0e18948f9e12e5160d008500",
+      "0x7323b13669028780c6450a620064e30654a5be2c",
+      "0xc0f2c8d014163ee1deaf52dd3b0df14b9e73dd08",
+      "0x038c9fbb2ad77b0b09639edbbc3afb82651e4e57",
```

This line should not have ending ","
```suggestion
      "0x038c9fbb2ad77b0b09639edbbc3afb82651e4e57"
```

---

oops sorry!

---

fixed